### ISSUE TITLE
Semantics behind list

### ISSUE THREAD

**Issue by annevk** [WICG/event-timing#49]
https://wicg.github.io/event-timing/#considered-for-event-timing has a list of event types. How were these chosen?

**Comment by tdresser** [comment]
Our objective was to construct a list of the events which would be caused by directly by user input.
We should be clearer about this in the spec (and it's very possible we missed something!).

**Comment by annevk** [comment]
What about `pointerrawupdate` or any other new input event type? Would that require changes? If so, we should really abstract this somehow. The `inert` attribute @alice is working on would also benefit from this.

**Comment by tdresser** [comment]
Yeah, we should figure out how to abstract this somehow. I'm really not sure what the best approach to doing this is though. Would we just want to move the list to the DOMEvent spec?

**Comment by annevk** [comment]
Letting UI Events own it sounds like a good start, as "user interaction event types" or some such.

An alternative I somewhat prefer would be to set a specific internal slot on the Event object, "due to user interaction" or some such, at the time of firing. Specifications could then check for the value of this internal slot. New user interaction event specifications would have to set this bit appropriately in order to integrate with the ecosystem. This might be a bit more involved though as it'd require finding and updating all the relevant specifications, some of which don't have the appropriate infrastructure.

**Comment by tdresser** [comment]
The internal slot approach does seem preferable, that's a good thought.
